Editor/Resources/EditorWindow/GameLiftPlugin.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using AmazonGameLift.Editor;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Editor.Resources.EditorWindow
+{
+    public class GameLiftPlugin : UnityEditor.EditorWindow
+    {
+        [SerializeField] private Texture icon;
+
+        private VisualTreeAsset _mVisualTreeAsset;
+        private VisualElement _root;
+        private VisualElement _currentTab;
+        private List<Button> _tabButtons;
+        private List<VisualElement> _tabContent;
+        private readonly TextProvider _textProvider = TextProviderFactory.Create();
+
+        private const string TabContentSelectedClassName = "TabContent--selected";
+        private const string TabButtonSelectedClassName = "TabButton--selected";
+        private const string TabButtonClassName = "TabButton";
+        private const string TabContentClassName = "TabContent";
+
+        private static GameLiftPlugin GetWindow()
+        {
+            var inspectorType = Type.GetType("UnityEditor.GameView,UnityEditor.dll");
```

Comment:
Is this necessary? I looked at other code references and we are able to directly pull the current window using 

Example: 
```
EditorWindow.GetWindow<SettingsWindow>()
```

Reply:
Specifying a `Type` as the first parameter tells the editor where to attempt to dock the window, in this case the main game view